doc/symfony-cache-configuration.rst

```diff
@@ -168,7 +168,8 @@ options through the constructor:
 
     With Apache, you can do this for example in a ``.htaccess`` file:
```

Comment:
I guess this should end in `::` instead to make the snippet below a proper code sample.